From 7107dc1e7dad7d1cba76c079bb341786868c6b34 Mon Sep 17 00:00:00 2001
From: Christian Ambach <ambi@samba.org>
Date: Sun, 3 Apr 2016 05:16:45 +0200
Subject: [PATCH 1053/1096] s3:smbd/service apply some code formatting

reduce indentation in switch statement, obey 80 char line limit, use C99 bool

Signed-off-by: Christian Ambach <ambi@samba.org>
Reviewed-by: Jeremy Allison <jra@samba.org>
(cherry picked from commit cb3d4b111ff2466c05ba3fc559b1b354f17cca1d)
---
 source3/smbd/service.c | 52 ++++++++++++++++++++++++++++----------------------
 1 file changed, 29 insertions(+), 23 deletions(-)

diff --git a/source3/smbd/service.c b/source3/smbd/service.c
index 44c87ed..0c70250 100644
--- a/source3/smbd/service.c
+++ b/source3/smbd/service.c
@@ -181,6 +181,7 @@ bool set_conn_connectpath(connection_struct *conn, const char *connectpath)
 bool set_current_service(connection_struct *conn, uint16_t flags, bool do_chdir)
 {
 	int snum;
+	enum remote_arch_types ra_type;
 
 	if (!conn)  {
 		last_conn = NULL;
@@ -206,30 +207,35 @@ bool set_current_service(connection_struct *conn, uint16_t flags, bool do_chdir)
 	last_conn = conn;
 	last_flags = flags;
 
-	/* Obey the client case sensitivity requests - only for clients that support it. */
+	/*
+	 * Obey the client case sensitivity requests - only for clients that
+	 * support it. */
 	switch (lp_case_sensitive(snum)) {
-		case Auto:
-			{
-				/* We need this uglyness due to DOS/Win9x clients that lie about case insensitivity. */
-				enum remote_arch_types ra_type = get_remote_arch();
-				if (conn->sconn->using_smb2) {
-					conn->case_sensitive = false;
-				} else if ((ra_type != RA_SAMBA) && (ra_type != RA_CIFSFS)) {
-					/* Client can't support per-packet case sensitive pathnames. */
-					conn->case_sensitive = False;
-				} else {
-					conn->case_sensitive = !(flags & FLAG_CASELESS_PATHNAMES);
-				}
-			}
-			break;
-		case True:
-			conn->case_sensitive = True;
-			break;
-		default:
-			conn->case_sensitive = False;
-			break;
-	}
-	return(True);
+	case Auto:
+		/*
+		 * We need this uglyness due to DOS/Win9x clients that lie
+		 * about case insensitivity. */
+		ra_type = get_remote_arch();
+		if (conn->sconn->using_smb2) {
+			conn->case_sensitive = false;
+		} else if ((ra_type != RA_SAMBA) && (ra_type != RA_CIFSFS)) {
+			/*
+			 * Client can't support per-packet case sensitive
+			 * pathnames. */
+			conn->case_sensitive = false;
+		} else {
+			conn->case_sensitive =
+					!(flags & FLAG_CASELESS_PATHNAMES);
+		}
+	break;
+	case True:
+		conn->case_sensitive = true;
+		break;
+	default:
+		conn->case_sensitive = false;
+		break;
+	}
+	return true;
 }
 
 /****************************************************************************
-- 
2.9.0

